docs/pages/blog/2021-developer-survey-results.md

```diff
@@ -0,0 +1,406 @@
+---
+title: 2021 MUI Developer Survey results
+description: Results for our yearly developer survey, 2021 edition.
+date: 2022-02-22T00:00:00.000Z
+authors: ['danilo-leal']
+tags: ['Developer survey']
+---
+
+Keeping up with the tradition, a few months ago we opened the 2021 MUI Developer Survey, to which we received 1,589 responses, 7% more answers than last year. This time, we have almost doubled the survey size, adding more depth to each covered topic, especially opening space for more open-ended questions. We wholeheartedly appreciate everyone who contributed, you are all helping shape the future of MUI as a company, so, thank you!
+
+Just like the previous iterations, the survey was chopped into three sections: ["Your needs"](#your-needs), ["Your product"](#your-product), and ["About you"](#about-you).
+
+## Your needs
+
+### How would you feel if you could no longer use MUI?
+
+Fortunately, and similar to 2019 and 2020, over 93% of the respondents would be disappointed if they could no longer use MUI (a sum of _"Very disappointed"_ with _“Somewhat disappointed”_). A very slight 1% decrease from 2020 (94%) but still as encouraging as it can be. MUI’s team will continue to grow to keep you all even more satisfied.
+
+<img src="/static/blog/2021-survey/1.png" style="width: 796px; margin-top: 16px; margin-bottom: 8px;" alt="Bar chart: 62.7% Very disappointed, 30.4% Somewhat disappointed, 6.9% Not disappointed." />
+
+<p class="blog-description">1567 out of 1589 answered</p>
+
+On the flip side though, the amount of "Very disappointed” alone fell by 10,3%, and the number of "Not disappointed" grew by 1,1%, in comparison with 2020. When choosing the last option, respondents were able to write a free-form answer to justify the choice. Based on the data, here are the main reasons we noticed and possible explanations for why these came to be:
```

Comment:
Suggestions for tightening up the text. I would also eliminate the decimals in the percentages as I don't think we need to be that precise here, or at least opt for the US English style of periods over commas: 10.3%.

```suggestion
Interestingly, the percentage who would be "Very disappointed" to stop using MUI fell by 10%, while those who would be "Not disappointed" grew by 1%. Those who chose the latter option were asked to explain why. Here are the biggest reasons we observed:
```